components/Form/TypeAheadWithOther/TypeAheadWithOther.tsx

```diff
@@ -41,9 +43,20 @@ const TypeAheadWithOther = ({
   className,
   suggestions,
   onSearch,
+  isRequired=false,
+  isRequiredVisualOnly=false,
   otherText = "Other",
 }: TypeAheadInputProps) => {
 
```

Comment:
@jupiter007 I have a weird issue with the translation code below. See the comments in the code. I commented out the non-working code for now, since it was breaking tests, and causing linting errors.

I don't understand why the translation doesn't work here. If you compare it to `FormInput`, I use it exactly the same way, but this one causes an error. (The only difference is that I named this one differently while debugging, since I thought it was conflicting with something else).